src/Domain/Fixture/FixtureCreator.php

```diff
@@ -915,4 +916,17 @@ private function displayStatus(): void {
       ->render();
   }
 
+  /**
+   * Check for security vulnerability advisories for installed packages.
+   */
+  private function auditPackages(): void {
+    $this->output->section('Running Composer Audit');
```

Comment:
Let's make this more "intention revealing", i.e., saying what the user cares about, not what's happening under the hood (which they can see in the log output, if they really need to know). Maybe something like this:

```suggestion
    $this->output->section('Showing security vulnerability advisories');
```